### PR TITLE
Initial release of role

### PR DESCRIPTION
**Pull request by UnderGreen**

### CODE REVIEW

File: handlers/main.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+
+- name: systemd daemon-reload
+  command: systemctl daemon-reload
```
Comment (rvadim) [inline handlers/main.yml]: You can use reload for one service:

```
systemctl reenable <service>
```

File: templates/etc/systemd/system/prometheus.service.j2 [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Prometheus
+After=network.target
+
+[Service]
+Type=simple
```
Comment (rvadim) [inline templates/etc/systemd/system/prometheus.service.j2:6]: Do you completly sure that, Type=simple is correct for Prometheus binary?

File: templates/etc/prometheus/prometheus.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    scrape_timeout:  {{ prometheus_global_evaluation_interval }}
+    static_configs:
+      - targets: ['localhost:9090']
+
+  - job_name: "node"
+    file_sd_configs:
+      - files:
+        - '{{ prometheus_file_sd_config_dir }}/*.json'
+        - '{{ prometheus_file_sd_config_dir }}/*.yml'
+        - '{{ prometheus_file_sd_config_dir }}/*.yaml'
```
Comment (rvadim) [inline templates/etc/prometheus/prometheus.yml.j2]: I think we need "other" sections for this config:

```
{% for job in jobs %}
  - job_name: "job.name"
    {{ job.config_type }}:
     ....
{% endfor %}
```

File: templates/etc/prometheus/prometheus.yml.j2 [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+scrape_configs:
+
+  - job_name: 'prometheus'
+    scrape_interval: {{ prometheus_global_scrape_interval }}
+    scrape_timeout:  {{ prometheus_global_evaluation_interval }}
+    static_configs:
+      - targets: ['localhost:9090']
+
+  - job_name: "node"
+    file_sd_configs:
```
Comment (rvadim) [inline templates/etc/prometheus/prometheus.yml.j2]: Actually we need many jobs for file_sd_configs due to call different namespaces and scrap intervals for different projects

File: templates/etc/prometheus/prometheus.yml.j2 [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  {% for dest in prometheus_rule_files %}
+  - {{ prometheus_rules_path }}/{{ dest }}
+  {% endfor %}
+{% endif %}
+
+# A list of scrape configurations.
+scrape_configs:
+
+  - job_name: 'prometheus'
+    scrape_interval: {{ prometheus_global_scrape_interval }}
```
Comment (rvadim) [inline templates/etc/prometheus/prometheus.yml.j2]: s/prometheus_global_scrape_interval/prometheus_self_scrape_interval/

File: handlers/main.yml (hunk 1 above)
Comment (UnderGreen) [inline handlers/main.yml]: Yes, sure. Thanks.

**Comment by rvadim** [comment]
Very good job overall :+1:

File: templates/etc/systemd/system/prometheus.service.j2 (hunk 2 above)
Comment (UnderGreen) [inline templates/etc/systemd/system/prometheus.service.j2:6]: I did't find information about forking main process. I think simple type is normal.

**Comment by UnderGreen** [comment]
@rvadim I updated role. Please review again.

File: defaults/main.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+prometheus_rule_files: []
+
+prometheus_web_external_url: "http://localhost:9090/"
+
+prometheus_config_flags:
+  'config.file': '{{ prometheus_config_dir }}/prometheus.yml'
+  'storage.local.path': '{{ prometheus_db_dir }}'
+  'web.external-url': '{{ prometheus_web_external_url }}'
+
+prometheus_jobs: []
```
Comment (rvadim) [inline defaults/main.yml:61]: Let's do some docs for this and simple example, just in comments

**Pull request merged**